src/io.ts

```diff
@@ -24,18 +25,6 @@ import {
   RGB,
 } from './utils';
 
-function getAxisLabels(arr: ZarrArray, axis_labels?: string[], channel_axis?: number) {
```

Comment:
Ah, I think `channel_axis` got lost when moving this to `src/utils.ts`. This is important for images without any dimension labels, as you can tell `vizarr` to separate out the channels in that dim like in napari.